src/aoc2020/day3_copy.clj

```diff
@@ -0,0 +1,112 @@
+(ns aoc2020.day3-copy
+  (:require [aoc2020.util :refer [file->seq]]))
+
+(defn tree-at-pos?
+  [grid position]
+  (let [row      (first position)
+        col      (second position)
+        location (-> row
+                     grid
+                     (get col))]
+    (= \# location)))
+
+(defn position-s
+  [grid slope-dist]
+  (let [row-number (count grid)
+        col-number (count (first grid))
+        row-s      (->> (iterate (fn [row]
+                                   (+ row (slope-dist :down)))
+                                 0)
```

Comment:
avoid iterate

Reply:
> avoid iterate

Could I create lazy-sequence using `range`? But the `range` has start and end, it can not be `inifinite`.

Reply:
range can be infinite